docs/newbs_flashing.md

```diff
@@ -12,23 +12,31 @@ However, the QMK Toolbox is only available for Windows and macOS currently.  If
 
 Begin by opening the QMK Toolbox application. You'll want to locate the firmware file in Finder or Explorer. Your keyboard firmware may be in one of two formats- `.hex` or `.bin`. QMK tries to copy the appropriate one for your keyboard into the root `qmk_firmware` directory.
 
-?> If you are on Windows or macOS there are commands you can use to easily open the current firmware folder in Explorer or Finder.
+If you are on Windows or macOS there are commands you can use to easily open the current firmware folder in Explorer or Finder.
 
-?> Windows:
+#### Windows
 
-    start .
+```
+start .
+```
 
-?> macOS:
+#### macOS
 
-    open .
+```
+open .
+```
 
 The firmware file always follows this naming format:
 
-    <keyboard_name>_<keymap_name>.{bin,hex}
+```
+<keyboard_name>_<keymap_name>.{bin,hex}
+```
 
 For example, the `plank/rev5` with a `default` keymap will have this filename:
```

Comment:
Could you fix this typo, since you're here already?

```suggestion
For example, the `planck/rev5` with a `default` keymap will have this filename:
```

Reply:
Sure thing, done.